packages/swap/src/handlers/openSwapDepositChannel.ts

```diff
@@ -12,7 +12,13 @@ import ServiceError from '../utils/ServiceError';
 export default async function openSwapDepositChannel(
   input: z.output<typeof openSwapDepositChannelSchema>,
 ) {
-  if (!validateAddress(input.destChain, input.destAddress, isProduction)) {
+  if (
+    !validateAddress(
+      input.destChain,
+      input.destAddress,
+      process.env.CHAINFLIP_NETWORK as ChainflipNetwork,
```

Comment:
could do an zod env migration here

Reply:
yes this should be the next app we tackle 🙂